plugin/plugin_go18.go

```diff
@@ -1,4 +1,4 @@
-//go:build go1.8 && !windows && amd64 && !static_build && !gccgo
+//go:build go1.8 && !windows && amd64 && !static_build && !gccgo && !containerd_no_plugins
```

Comment:
```suggestion
//go:build go1.8 && !windows && amd64 && !static_build && !gccgo && !containerd_no_dynamic_plugins
```

Reply:
Works for me 👍 
I pushed a commit to update both files at once